tests/backup/backup_delete_test.go

```diff
@@ -670,3 +671,155 @@ var _ = Describe("{DeleteBucketVerifyCloudBackupMissing}", func() {
 		}
 	})
 })
+
+// DeleteBackupAndCheckIfBucketIsEmpty delete backups and verify if contents are deleted from backup location or not
+var _ = Describe("{DeleteBackupAndCheckIfBucketIsEmpty}", func() {
+	numberOfBackups, _ := strconv.Atoi(getEnv(maxBackupsToBeCreated, "10"))
+	var (
+		scheduledAppContexts     []*scheduler.Context
+		backupLocationUID        string
+		cloudCredUID             string
+		cloudCredUidList         []string
+		bkpNamespaces            []string
+		clusterUid               string
+		clusterStatus            api.ClusterInfo_StatusInfo_Status
+		customBackupLocationName string
+		credName                 string
+	)
+	timeBetweenConsecutiveBackups := 10 * time.Second
+	backupNames := make([]string, 0)
+	numberOfSimultaneousBackups := 4
+	labelSelectors := make(map[string]string)
+	bkpNamespaces = make([]string, 0)
+	backupLocationMap := make(map[string]string)
+	credMap := make(map[string]map[string]string)
```

Comment:
Looks like credMap is not being used anywhere.

Reply:
Ack removed